lib/msal-common/src/unifiedCache/entities/IdTokenEntity.ts

```diff
@@ -27,4 +28,29 @@ export class IdTokenEntity extends Credential {
 
         return idTokenKeyArray.join(Separators.CACHE_KEY_SEPARATOR).toLowerCase();
     }
+
+    /**
+     * Create IdTokenEntity
+     * @param homeAccountId
+     * @param authenticationResult
+     * @param clientId
+     * @param authority
+     */
+    static createIdTokenEntity(
```

Comment:
Is this class a replacement for the IdToken and IdTokenClaims class, or is it just used by the cache?

Reply:
Just used by the cache. `IdToken` and `IdTokenClaims` classes will remain.